services/backend-for-frontend-nodejs/package.json

```diff
@@ -10,18 +10,19 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@opentelemetry/api": "^1.8.0",
-    "@opentelemetry/auto-instrumentations-node": "^0.43.0",
-    "@opentelemetry/exporter-trace-otlp-proto": "^0.49.1",
-    "@opentelemetry/instrumentation-undici": "^0.2.0",
-    "@opentelemetry/sdk-node": "^0.49.1",
-    "express": "^4.18.3"
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/auto-instrumentations-node": "^0.58.1",
+    "@opentelemetry/exporter-trace-otlp-proto": "^0.200.0",
+    "@opentelemetry/resources": "^2.0.1",
+    "@opentelemetry/sdk-node": "^0.200.0",
+    "@opentelemetry/semantic-conventions": "^1.33.0",
```

Comment:
When I run `npm i @opentelemetry/semantic-conventions`, it automatically shows up in my `package.json` file like this.

Reply:
So there is nothing actually to change in the instructions.